src/test/java/spoon/test/textBlocks/TextBlockTest.java

```diff
@@ -0,0 +1,49 @@
+package spoon.test.textBlocks;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import spoon.Launcher;
+import spoon.reflect.code.CtInvocation;
+import spoon.reflect.code.CtLiteral;
+import spoon.reflect.code.CtStatement;
+import spoon.reflect.code.CtTry;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtMethod;
+
+public class TextBlockTest{
+	private Launcher setUpTest() {
+		Launcher launcher = new Launcher();
+		launcher.addInputResource("./src/test/resources/textBlock/TextBlockTestClass.java");
+		launcher.run();
+		return launcher;
+	}
+	
+	@Test
+	public void testTextBlock(){
+		Launcher launcher = setUpTest();
+		CtClass<?> allstmt = (CtClass<?>) launcher.getFactory().Type().get("textBlock.TextBlockTestClass");
+		CtMethod<?> m1 =  allstmt.getMethod("m1");
+		CtStatement stmt1 = m1.getBody().getStatement(0);
+		CtLiteral l1 = (CtLiteral) stmt1.getDirectChildren().get(1);
```

Comment:
maybe use TypeFilter for more stable. Get(Number) is pretty unstable for a testcase. If we change order in any future case